tests/foreman/api/test_docker.py

```diff
@@ -453,9 +452,38 @@ def test_add_multiple_docker_repos_to_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        product = entities.Product(organization=self.org).create()
+        repos_amount = randint(1, 5)
+        repos = [
+            _create_repository(product, name=gen_string('alpha'))
+            for _
+            in range(repos_amount)
+        ]
+        self.assertEqual(len(product.read().repository), repos_amount)
```

Comment:
My expression can catch situation when I wanted for example create 4 repos, but 3 was actually created/assigned